### PR TITLE
staging-wifi-1345

### PR DESCRIPTION
**Pull request by raj-TIP**
Added Perfecto pyTest

### CODE REVIEW

File: tests/e2e/interOp/pytest.ini [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[pytest]
```
Comment (jaspreetsachdev) [inline tests/e2e/interOp/pytest.ini:1]: @shivamcandela  should this information

Comment (shivamcandela) [reply tests/e2e/interOp/pytest.ini:1]: We need to remove this file and use the main pytest.ini

File: tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        driver = setup_perfectoMobileWeb[0]
+        connData = get_APToMobileDevice_data
+
+        #Set Wifi/AP Mode
+        #set_APconnMobileDevice_iOS(ssidName, ssidPassword, setup_perfectoMobile_iOS, connData)
+
+        #Verify Upload download Speed from device Selection
+        verifyUploadDownloadSpeediOS(setup_perfectoMobileWeb, connData)
+        #verifyUploadDownloadSpeediOSRemoteDriver(setup_perfectoMobileWeb, connData)
+
```
Comment (shivamcandela) [inline tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py:55]: Assert statement is missing here

File: tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        ssidPassword = profile_data["security_key"]
+        print ("SSID_NAME: " + ssidName)
+        print ("SSID_PASS: " + ssidPassword)
+
+        report = setup_perfectoMobileWeb[1]
+        driver = setup_perfectoMobileWeb[0]
+        connData = get_APToMobileDevice_data
+
+        #Set Wifi/AP Mode
+        #set_APconnMobileDevice_iOS(ssidName, ssidPassword, setup_perfectoMobile_iOS, connData)
```
Comment (shivamcandela) [inline tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py:50]: this line needs to be uncommented?

File: tests/e2e/interOp/android/BaseClassAndroid.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+import sys
+import unittest
+from unittest.loader import getTestCaseNames
+import warnings
+from perfecto.model.model import Job, Project
+import urllib3
+import argparse
+from perfecto import (PerfectoExecutionContext, PerfectoReportiumClient,TestContext, TestResultFactory)
+from appium import webdriver
+#rom selenium import webdriver
```
[the remark this answers is not in the document]
Comment (raj-TIP) [reply tests/e2e/interOp/android/BaseClassAndroid.py]: File removed

File: tests/e2e/interOp/android/BaseClassAndroid.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+import pytest
```
[the remark this answers is not in the document]
Comment (raj-TIP) [reply tests/e2e/interOp/android/BaseClassAndroid.py]: file removed as its the old Android tests

File: tests/e2e/interOp/android/BaseClassAndroid.py [hunk 6]
```diff
@@ -0,0 +1,129 @@
+import os
```
[the remark this answers is not in the document]
Comment (raj-TIP) [reply tests/e2e/interOp/android/BaseClassAndroid.py]: File removed

File: tests/e2e/interOp/android/BaseClassAndroid.py [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+import warnings
+from perfecto.model.model import Job, Project
+import urllib3
+import argparse
+from perfecto import (PerfectoExecutionContext, PerfectoReportiumClient,TestContext, TestResultFactory)
+from appium import webdriver
+#rom selenium import webdriver
+import pytest
+
+class TestConf(unittest.TestCase):
```
[the remark this answers is not in the document]
Comment (raj-TIP) [reply tests/e2e/interOp/android/BaseClassAndroid.py]: file removed as its the old android scripts

File: tests/e2e/interOp/pytest.ini (hunk 1 above)
Comment (raj-TIP) [reply tests/e2e/interOp/pytest.ini]: fixed/updated

File: libs/perfecto/README.md [hunk 8]
```diff
@@ -1 +1,2 @@
 ## Perfecto libraries
+#Initial Check in - Raj
```
[the remark this answers is not in the document]
Comment (raj-TIP) [reply libs/perfecto/README.md]: Added few purest installation.  Will be updating this file after android scripts is committed.

File: tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py (hunk 2 above)
Comment (raj-TIP) [reply tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py:55]: fixed updated, it was a test fixture for internal testing. and its removed.

File: tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py (hunk 3 above)
Comment (raj-TIP) [reply tests/e2e/interOp/iOS/client_connectivity/test_NatMode.py:50]: No, this file was used for testing and I fixed it the latest release.

**Review by shivamcandela — APPROVED** [review]

**Review by shivamcandela — APPROVED** [review]

**Review by shivamcandela — APPROVED** [review]

**Review by shivamcandela — APPROVED** [review]

**Pull request merged**